skbio/sequence/_dna.py

```diff
@@ -222,6 +220,7 @@ def translate(self, *args, **kwargs):
 
         See Also
         --------
+        RNA.reverse_transcribe
```

Comment:
I would vote to keep, these are all related to the central dogma (roughly). Talked with @jairideout offline and he's fine with keeping it also.